packages/driver/src/cypress/log.ts

```diff
@@ -92,14 +92,12 @@ export const LogUtils = {
 
     return _
     .chain(tests)
-    .flatMap((test) => {
-      return [test, test.prevAttempts]
-    })
-    .flatMap<{id: number}>((tests) => {
-      return [].concat(tests.agents, tests.routes, tests.commands)
-    }).compact()
-    .union([{ id: 0 }])
-    .map('id')
+    .flatMap((test) => test.prevAttempts ? [test, ...test.prevAttempts] : [test])
+    .flatMap<{id: string}>((tests) => [].concat(tests.agents, tests.routes, tests.commands))
+    .compact()
+    .union([{ id: '0' }])
+    // id is a string in the form of 'log-origin-#', grab the number off the end.
```

Comment:
Why do the origin logs need to be name-spaced?

Reply:
Yes, it's the best way to guarantee each log has a unique ID, since any counter used will be reset in each version of the driver and it's more work than it's worth to try to sync the counter.